modules/ROOT/pages/websockets/websocket-connector-server-components.adoc

```diff
@@ -0,0 +1,55 @@
+= About Websocket Server Components
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: connectors, ws, wss, websocket
+
+== About Websocket Server Components
+
+=== Inbound Listener
+
+The Inbound Listener component represents an websocket endpoint that clients can connect to. By hitting this endpoint, external clients not only establish a Websocket connection, they will trigger the owning server flow each time they send a message.
+
+[source,xml,linenums]
+----
+<flow name="onIncomingMessage">
+    <websocket:inbound-listener ​path​=​"/quotes"​ ​defaultGroups​=​"#[['AVENGERS', 'REVENGERS']]"​ ​config-ref​=​"ws"​ />
+    <set-variable name="myMessage" value="#[payload]" />
+    <set-variable name="senderSocketId" value="#[attributes.socketId]" />
+    <set-variable name="senderSocketType" value="#[attributes.socketType]" />
+    <set-variable name="senderSocketGroups" value="#[attributes.groups]" />
+</flow>
+----
+
+The payload will be set with the received message content, and its mime-type will be obtained from the incoming HTTP request header. If not set, the mime-type will default to one of:
+
+* application/octet-stream if it's binary content.
+* text/plain if it's text content.
+
+The output attributes will contain every attribute expected in the HttpRequestAttributes plus:
+
+    * Receiving socket's id.
+    * Receiving socket's type.
+    * Group list the receiving socket is currently subscribed to.
+
+=== On Inbound Connection
```

Comment:
this fails to explain that if this flow fails, then the connection will be dropped, and that all messages will be queued until this flow ends.

It also doesn't explain that you cannot have two of these sources listening for the same path, nor it shows a useful usecase for this, like validating a header or updating some state.